modules/n1ql/pages/n1ql-language-reference/selectclause.adoc

```diff
@@ -1,4 +1,5 @@
 = SELECT Clause
+:imagesdir: ../../assets/images
```

Comment:
One question: how are the images above created? The output in the preview looks good.

This commit has a lot of Asciidoc features I don't know enough to comment on, will study...

Reply:
[Railroad Diagram Generator](https://www.bottlecaps.de/rr/ui) by Gunther Rademacher. There's a standalone tool as well as the online converter.